Frontend/ui-driver/index.js

```
@@ -457,36 +401,40 @@ window["onEvent"] = function (jsonPayload, args, callback) { // onEvent from hyp
+  window.Android.runInUI("android.webkit.WebView->setWebContentsDebuggingEnabled:b_false;", "null");
 }
 
-console.log("APP_PERF INDEX_BUNDLE_END : ", new Date().getTime());
\ No newline at end of file
+JBridge.runInJuspayBrowser("onEvent", JSON.stringify({
+  action: "DUI_READY",
+  event: "initiate",
+  service: JOS.self
+}), "");
+
+eval(window.JBridge.loadFileInDUI("v1-tracker.jsa"));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)